src/index.html

```diff
@@ -0,0 +1,69 @@
+<html>
+  <head>
+    <link rel="stylesheet" href="css/vote-CSS.css" />
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css2?family=Varela+Round&display=swap'"
+    />
+  </head>
+  <body>
+    <div class="test_box">
+      <div class="u-layout">
+        <img class="u-layout" src="c:\Pics\logo.png" alt="logo" />
+      </div>
+
+      <div class="u-layout">
+        <h1>Welcome Back</h1>
+      </div>
+      <div class="u-layout">
+        <img id="google_logo" src="c:\Pics\google.png" alt="google_logo" />
```

Comment:
An `alt` tag is the alternative description of the image for screen readers, so it needs to be text. It would be something like `Google company logo`